lizmap/modules/lizmap/lib/Project/Project.php

```diff
@@ -1258,22 +1267,65 @@ public function getLoginFilters($layers, $edition = false)
             // attribute to filter
             $attribute = $loginFilteredConfig->filterAttribute;
 
+            // Quoted attribute with double-quotes
+            // $cnx = $this->appContext->getDbConnection();
+            $cnx = \jDb::getConnection();
```

Comment:
@mdouchin  you must not use `\jDb::getConnection();`.

To ensure that tests pass, `ContextForTests::getDbConnection()` should return `new jDbConnectionForTests()`.